### PR TITLE
Changed props defaultDate and miniumumDate to expect a Unix timestamp

### PR DESCRIPTION
**Pull request by tumentumurchudur**
Fixes https://github.com/mxenabled/mx-react-components/issues/89
Changed props defaultDate and miniumumDate to expect a Unix timestamp.
Cleaned up the unix date conversion code.

### CODE REVIEW

File: src/components/DatePicker.js [hunk 1]
```diff
@@ -21,14 +21,14 @@ class DatePicker extends React.Component {
       this._handleBlur();
     }
     this.setState({
-      selectedDate: date
+      selectedDate: moment.unix(date)
     });
 
     this.props.onDateSelect(moment.unix(date).format(this.props.format));
```
Comment (jmophoto) [inline src/components/DatePicker.js]: If we send the component a unix timestamp, we should return the date in the same format.

Comment (tumentumurchudur) [inline src/components/DatePicker.js]: yes, I agree. :+1:

**Comment by tumentumurchudur** [comment]
Selected date is returned in the Unix date format.

**Comment by bsbeeks** [comment]
This looks good.

**Pull request merged**